ambry-frontend/src/test/java/com.github.ambry.frontend/AmbryBlobStorageServiceTest.java

```diff
@@ -998,6 +1000,28 @@ private void setAmbryHeadersForPut(JSONObject headers, long ttlInSecs, boolean i
     }
   }
 
+  /**
+   * Sets headers for conditional delete.
+   * @param headers the {@link JSONObject} where the headers should be set.
+   * @param targetAccountName sets the {@link RestUtils.Headers#TARGET_ACCOUNT_NAME} header. Can be {@code null}.
+   * @param targetContainerName sets the {@link RestUtils.Headers#TARGET_CONTAINER_NAME} header. Can be {@code null}.
+   * @throws IllegalArgumentException if {@code headers} is null.
+   * @throws JSONException
+   */
+  private void setAmbryHeadersForDelete(JSONObject headers, String targetAccountName, String targetContainerName)
```

Comment:
minor: call this `setAccountAndContainerHeaders()` - can be reused for non deletes also if required